src/actions/index.js

```diff
@@ -1,9 +1,14 @@
 export const RECEIVE_FORECAST = 'RECEIVE_FORECAST'
 
 export const receiveForecast = (forecastData) => ({
-  // action object
+  type: RECEIVE_FORECAST,
+  forecastData,
+  data: do something to the weater data
 });
 
-export const fetchForecast = options => dispatch => {
-  // return fetch API call
+export const fetchZipForecast = forecastType, zip, key => dispatch => {
```

Comment:
🤗 
But I think we should pull the 'key' variable back outside the function, where it makes sense to live as a thing that gets passed in, no? (The thinking was if the key ever expires, it'll only need to be updated in one place.)